app/components/Form/ApplicationForm.tsx

```diff
@@ -474,30 +499,29 @@ const ApplicationForm: React.FC<Props> = ({
         typeof newFormSectionData?.firstNationsLed !== 'undefined' &&
         firstNationsLed !== jsonData.projectArea?.firstNationsLed;
 
-      if (isSubmitted && !projectAreaAccepted) {
-        // revert form data
-        newFormData = {
-          ...jsonData,
-        };
-        if (geographicAreaInputChanged && isZoneSpecificIntake) {
-          // display new modal saying
-          // Invalid selection. You have indicated that this project is not led or supported by First Nations, therefore, you may only choose from zones 1,2,3 or 6.
-          setProjectAreaModalType('invalid-geographic-area');
+      const projectAreaAccepted = isZoneSelectionValid(
```

Comment:
Should stick to the `is` prefix for booleans
```suggestion
      const isProjectAreaAccepted = isZoneSelectionValid(
      //could also do isValidZoneSelection
```